backend/src/github.ts

```diff
@@ -35,59 +35,62 @@ const getUser = async (token: string): Promise<GitHubUser> => {
   }
 }
 
-const createFork = async (
-  token: string,
-  newForkName: string,
-  template: ForkTemplate
-): Promise<ExtendedFork> => {
```

Comment:
Our GitHub module should not know anything about our own API's models (`ExtendedFork`).

Reply:
Syy?

Reply:
ExtendedFork on meidän APIn palauttama tietomalli, josta GitHub-moduulin ei tarvis tietää mitää. GitHub-moduuli tietää mitä sen oma API palauttaa ja sit ois fork-handlerin vastuulla palauttaa mitä ollan sovittu.